### DISCUSSION TITLE
Forced 2FA is wrong

### DISCUSSION THREAD

**Discussion by KOYK** [community/community#73312] [Other Feature Feedback, Questions, & Ideas]
I don't like being forced to do things. The fact that the github\microsoft people think that they know what is good for me and what is not, and somehow they have the right to force it on me, assuming that I can't figure how to do things, is really really really frustrating to say the least. I really feel that im being oppressed, especially when you lock me out of my account until I do your bidding.

**Comment by AwokeKnowing** [comment]
you are correct. Microsoft does not care about your security. They care about control. They want to be able to prove who uploaded/edited code in order to turn you over to the government. and they also have incompetent project managers who do not produce anything but need to show progress and so they just bully developers and ignore the community.

This is no longer an appropriate place to host open source projects.
